internal/host/instructionmanager_test.go

```diff
@@ -192,6 +195,7 @@ func checkStepsByState(state string, host *models.Host, db *gorm.DB, mockEvents
 		{DriveType: "disk", Name: "sdh", SizeBytes: validDiskSize},
 	}
 	mockValidator.EXPECT().GetHostValidDisks(gomock.Any()).Return(disks, nil).AnyTimes()
+	mockRelease.EXPECT().GetMCOImage(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return("mcoImage", nil).AnyTimes()
```

Comment:
Bad habit to use `AnyTimes` and probably need to add a case with a failure.

Reply:
This is inside a generic helper method to check state, that is called from many testcases.